bot/exts/evergreen/snakes/_utils.py

```diff
@@ -461,7 +461,7 @@ async def _add_player(self, user: Member) -> None:
         self.players.append(user)
         self.player_tiles[user.id] = 1
 
-        avatar_bytes = await user.avatar_url_as(format="jpeg", size=PLAYER_ICON_IMAGE_SIZE).read()
+        avatar_bytes = await user.display_avatar.read()
```

Comment:
Should we still replace this with `PLAYER_ICON_IMAGE_SIZE`?

Reply:
Indeed, indeed, forgot about that one